content/contributing/helping-out/reviewing-pull-requests.md

```diff
@@ -0,0 +1,60 @@
+
+## How to review a Pull Request
+
+If you're new to GitHub, there's a lot of great information on the official [Pull Request Review documentation](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews). If you're happy with the work and feel you're reasonably qualified to assess quality in this particular area, leave your Approved review on the _PR_. Again, anyone can and should leave reviews ... no special permissions are required!
+
+## Giving feedback
+
+Focus on giving constructive, actionable feedback that results in real improvements to code quality or end-user experience. If you don't understand why an approach was taken, please ask!
+
+Provide actual code suggestions when that is helpful. Small changes work well as comments or in-line suggestions on specific lines of codes. Larger changes deserve a comment in the main thread, or a pull request to the original author's branch (but please mention that you've made one). When in doubt about a matter of architectural philosophy, refer back to [What we're trying to build](@/contributing/introduction.md#what-we-re-trying-to-build) for guidance.
+
+It's okay to leave an approval even if you aren't 100% confident on all areas of the PR: just be sure to note your limitations. When maintainers are evaluating the PR to be merged, they'll make sure that there's good coverage on all of the critical areas. If you can only check that the math is correct, and another reviewer can check everything but the math, we're in good shape!
+
+Similarly, if there are areas that would be good to fix but aren't severe, please consider leaving an approval. The author can address them immediately, or spin it out into follow-up issues or PRs. Large PRs are much more draining for both reviewers and authors, so try to push for a smaller scope with clearly tracked follow-ups.
+
+## Requesting changes
+
+It is quite common for reviews to ask for additional changes, but as a reviewer it's important to be respectful of the author's time.
+
+It is always acceptable to ask an author to fix an issue or bug in their code, or request they do something differently. It is also fine to request additional documentation or tests, but when you do it's best to outline what exactly is lacking in the current version.
+
+Otherwise please try to avoid asking for additional work unless absolutely necessary, especially if the PR is in an otherwise acceptable state. The purpose of an open source code review is to assess the code as it is, not to tell the author what to do.
```

Comment:
```suggestion
Otherwise, please try to avoid asking for additional work unless absolutely necessary, especially if the PR is in an otherwise acceptable state. The purpose of an open source code review is to assess the code as it is, not to tell the author what to do.
```